### ISSUE TITLE
SyntaxError: import declarations may only appear at top level of a module

### ISSUE THREAD

**Issue by nascent** [CoderLine/alphaTab#723]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am getting the aforementioned error.

### Expected Behavior

No error.

### Steps To Reproduce

1. go to the following page: https://www.emotive.productions/notation/under-pandemic-skies/player/
2. Observe error in Developer Tools

### Found in Version

1.3-alpha

### Platform

Web, version 150

### Environment

```markdown
- **OS**: Windows 10
- **Browser**: FireFox

### Anything else?

There is no error, in Chrome.

**Comment by Danielku15** [comment]
@nascent I highly discourage using the alpha builds of alphaTab (phun intended) for your productive installation. I would recommend only consuming stable versions.

**Comment by nascent** [comment]
It is fixed in 1.3-alpha 153 (or below).
